### DISCUSSION TITLE
Phone input!

### DISCUSSION THREAD

**Discussion by jimmailcamp** [heroui-inc/heroui#4387] [Feature requests]
Localized phone input. No need to explain I think.:)
Would be dope. 

x

**Comment by Alexandregirbal** [comment]
I agree, it would be nice to have a phone input that enforces the ITU-T **E.164** format.
It would avoid results like `020 1234 5678` and standardize to `+442012345678`.
Backend validation would still be needed, but no more manual parsing.
The input could be divided in two parts as we can see on many websites:
- a country code selector with a flag picker
- the local number, with or without the leading 0
